docs/meshstack.how-to.get-started-with-meshstack.md

```diff
@@ -3,39 +3,32 @@ id: meshstack.how-to.get-started-with-meshstack
 title: How to get started with meshStack
 ---
 
-This how-to guide is targeted at Cloud Foundation Partners that are starting out with a brand new meshStack. It is intended to be the first resource you read. You need to be able to log into your meshStack to follow this guide.
+This a guide that accompanies you on your very first stepts setting up a Internal Developer Platform with meshStack.
 
-## Step 1: Log into your meshStack
+> **Prerequisites:**
+> 1. Your new meshStack was setup and the link to the meshPanel was provided to you.
+> 2. You need to be able to log into your meshStack to follow this guide.
 
-As a Cloud Foundation Partner you are directly forwarded to the admin area of meshStack. This is the cockpit that helps you stay in control of everything happening in meshStack.
+## Step 1: Access the Admin Area of Your meshStack
 
-This it what it looks like.
+The Admin Area is the central place to setup, customize and operate your very own Internal Developer Platform via meshStack.
+To return to this view at a later point in time, you can always press the "Admin Area" button in the top bar.
 
-![Admin Area](assets/admin-area.png)
+## Step 2: Create Your First Workspace
 
-To return to this view, you can always press the "Admin Area" button in the top bar.
+Workspaces provide you the opportunity to onboard and structure your use cases. Those workspaces can act as consumers and providers of services within the marketplace. The marketplace is 
+the central place in meshStack to offer services that others consume. 
 
-If you are not able to see this screen, you are missing partner rights. This can be fixed by any other partner user.
-
-## Step 2: Create your first meshWorkspace
-
-The workspaces of your Cloud Foundation are represented by meshStack as meshWorkspaces.
-
-To experience what this feels like, create your first workspace via the panel. To open the wizard, click on "Create Workspace +" as shown in the picture below.
+Now lets get started by creating the first workspace in your meshStack. To open the wizard, click on "Create Workspace +" as shown in the picture below.
 
 ![Create meshWorkspace](assets/create-workspace.png)
 
 Of course, this can also be achieved via API. Take a look at our [workspace API](/api/index.html#_meshworkspace) to learn more.
```

Comment:
As cool as this is, this should not be a part of a "Getting started" guide. Let's remove it.

Reply:
agreed!